cscs-checks/apps/jupyter/check_ipcmagic.py

```diff
@@ -12,7 +12,8 @@ def __init__(self):
         self.modules = ['ipcmagic']
         self.pre_run = [
             'module unload dask',
```

Comment:
Why this unload is needed? Is it loaded by default, because the test doesn't seem to load it.

Reply:
yes, it's loaded automatically by jupyterlab, which is loaded by ipcmagic. The problem is that TensorFlow also loads dask but a different version than the one loaded by jupyterlab. The unload is to avoid the conflict error.